modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -290,6 +290,28 @@ TEST_P(Dilate3x3Test, AccuracyTest)
     }
 }
 
+TEST_P(MorphologyExTest, AccuracyTest)
+{
+    int defShape = cv::MORPH_RECT, defKernSize = 3;
```

Comment:
Could it be a test parameter?

Reply:
Can these parameters have another values?

Reply:
Yes, they can have a lot. But I decided to hardcode only one not to make a lot of test instances. There is a possibility for that because these params are tested in dilate/erode kernels tests (above), and morphologyEx is in fact a sequence of dilate/erode so there is no need to re-test them

Reply:
Ok.